### PR TITLE
Discogs service

### PR DESCRIPTION
**Pull request by pedroCervi**
Hello!

Here we add VCR configuration and some tests to a secret discogs service.

### CODE REVIEW

File: spec/services/discogs_requests_service_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe DiscogsRequestsService, type: :service do
+  describe '#search_artist_releases' do
+    it 'gets an artist releases sussessfully' do
+      VCR.use_cassette("get_artist_the_who_releases") do
+        response = described_class::Client.new.search_artist_releases('The Who')
+
+        expect(response.first.title).to eq("I Can't Explain")
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: Single quotes :nail_care:

File: spec/services/discogs_requests_service_spec.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      end
+    end
+  end
+
+  describe '#get_artist' do
+    it 'gets an artist sussessfully' do
+      VCR.use_cassette("get_artist_the_who") do
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: Single quotes :nail_care:

File: spec/services/discogs_requests_service_spec.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      VCR.use_cassette("get_artist_the_who_releases") do
+        response = described_class::Client.new.search_artist_releases('The Who')
+
+        expect(response.first.title).to eq("I Can't Explain")
+      end
+    end
+  end
+
+  describe '#get_artist' do
+    it 'gets an artist sussessfully' do
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: Typo: ~sussessfully~ successfully :nail_care:

File: spec/services/discogs_requests_service_spec.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe DiscogsRequestsService, type: :service do
+  describe '#search_artist_releases' do
+    it 'gets an artist releases sussessfully' do
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: Typo: ~sussessfully~ successfully :nail_care:

File: spec/services/discogs_requests_service_spec.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      VCR.use_cassette("get_artist_the_who_releases") do
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: Single quotes :nail_care:

Comment (izn) [reply spec/services/discogs_requests_service_spec.rb]: Also, I think we could use subdirectories here. Something like this:

```ruby
VCR.use_cassette('services/discogs/get_artist')
```

IIRC this should work.

File: spec/services/discogs_requests_service_spec.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  describe '#get_artist' do
+    it 'gets an artist sussessfully' do
+      VCR.use_cassette("get_artist_the_who") do
+        response = described_class::Client.new.get_artist('The Who')
+
+        expect(response.name).to eq('The Who')
+      end
+    end
+  end
+end
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb:49]: By the way, should we create an expectation block to cover not found artists?

File: spec/rails_helper.rb [hunk 7]
```diff
@@ -67,3 +69,8 @@
   # Include login
   config.include Devise::Test::ControllerHelpers, type: :controller
 end
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
```
Comment (millnitzluan) [inline spec/rails_helper.rb]: Single quotes

File: spec/services/discogs_requests_service_spec.rb [hunk 8]
```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe DiscogsRequestsService, type: :service do
+  describe '#search_artist_releases' do
+    it 'gets an artist releases sussessfully' do
+      VCR.use_cassette("get_artist_the_who_releases") do
+        response = described_class::Client.new.search_artist_releases('The Who')
```
Comment (izn) [inline spec/services/discogs_requests_service_spec.rb]: We could use a global `subject`.

```ruby
subject(:discogs_service) { DiscogsRequestsService.new }

describe '#search_artist_relases' do
  it 'something' do 
    response = discogs_service.search_artist_releases('The Who')

    ...
  end
end

describe '#get_artist' { ... }
```

WDYT?

File: spec/services/discogs_requests_service_spec.rb (hunk 6 above)
Comment (pedroCervi) [reply spec/services/discogs_requests_service_spec.rb:49]: Cara é muito dificil não existir no discogs. Mas coloquei um random ali, deve retornar um hash vazio daí.

**Review by izn — APPROVED** [review]
LGTM :tada:

**Review by millnitzluan — APPROVED** [review]

**Pull request merged**